Main/Source/item.cpp

```diff
@@ -1918,11 +1918,11 @@ void itemlock::PostConstruct()
 
 truth itemlock::TryKey(item* Key, character* Applier)
 {
-  if(GetVirtualConfig() & BROKEN_LOCK)
```

Comment:
May I suggest this #352 as a fix for this instead?

Reply:
Nice, I'll revert my change.